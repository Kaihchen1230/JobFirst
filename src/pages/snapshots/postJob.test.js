import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import PostJob from "../postJob"
//import "../../style/postJob.css";

describe("postJob snapshot test", () => {
    test('postJob render should match snapshot', () => {
        const render = new ReactShallowRenderer();
        render.render(<PostJob />);
        expect(render.getRenderOutput()).toMatchSnapshot();
    });
});