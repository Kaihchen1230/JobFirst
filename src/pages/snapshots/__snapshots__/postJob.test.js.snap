// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correct output of postJob page 1`] = `
<div align="center">
                <br />
                <h1>Post a New Job</h1>
                <Form onSubmit={this.submitData} className="main-form">
                    <Form.Item>
                        <Input placeholder="Enter Employer Name"
                            prefix={<Icon type="user" />}
                            suffix={
                                <Tooltip title="Enter the name of the employer.">
                                    <Icon type="info-circle" />
                                </Tooltip>}
                        />
                    </Form.Item>
                    <Form.Item>
                        <DatePicker placeholder="Date Posted On" />
                    </Form.Item>
                    <Form.Item>
                        <Select placeholder="Job Type">
                            <Option value="Full Time">Full Time</Option>
                            <Option value="Part Time">Part Time</Option>
                            <Option value="Internship">Internship</Option>
                            <Option value="Temporary">Temporary</Option>
                        </Select>
                    </Form.Item>
                    <Form.Item>
                        <TextArea
                            placeholder="Enter Job Description" autosize={{ minRows: 2, maxRows: 6 }}
                        />
                    </Form.Item>
                    <Form.Item>
                        <Input placeholder="(###) ###-####"
                            prefix={<Icon type="contacts" />}
                            suffix={
                                <Tooltip title="Enter the contact number of the employer.">
                                    <Icon type="info-circle" />
                                </Tooltip>}
                        />
                    </Form.Item>
                    <Form.Item>
                        <Button type="primary" htmlType="submit">Submit Job</Button>
                    </Form.Item>
                </Form>
            </div>
`;
